client/webpack.config.js

```diff
@@ -378,7 +378,7 @@ module.exports = (env = {}, argv = {}) => {
                 allowAsyncCycles: false,
                 // set the current working directory for displaying module paths
                 cwd: process.cwd(),
-            }),
+            })
```

Comment:
```suggestion
            }),
```
this was prettier on my local branch 😂